config/locales/en.yml

```diff
@@ -40,3 +40,11 @@ en:
           friend at %{invitation_page}.
           Cheers, 
           %{archive_name}
+    coauthor_notification:
+      subject: "[%{app_name}] Co-Author Notification"
+      edit_the: "Edit the %{type}"
```

Comment:
And a bigger complication - what about languages in which, say, work is male and chapter is female, or something, so "the" changes? They'll need multiple options here.